libraries/hatch-server-middleware/package.json

```diff
@@ -24,7 +24,12 @@
     "express": "^4",
     "utf-8-validate": "^5",
     "winston": "^3",
-    "ws": "^7"
+    "ws": "^7",
+    "@types/cookie": "0.4.1",
```

Comment:
peer dependencies should also be dev dependencies

Reply:
updated to dependencies